rebar.config

```
@@ -8,14 +8,54 @@
 {xref_checks, [undefined_function_calls, undefined_functions,
                locals_not_used, deprecated_function_calls,
                deprecated_functions]}.
-
+{profiles, [
+  {brod_cli, [
+    {deps, [ {docopt, {git, "https://github.com/zmstone/docopt-erl.git", {branch, "0.1.3"}}}
+           , {jsone, "1.7.0"}
+    ]},
+    {erl_opts, [{d, build_brod_cli}]},
+    {escript_name, brod_cli},
+    {relx, [{release, {brod, "i"}, % release the interactive shell as brod-i
+      [brod, jsone, docopt]},
+      {include_erts, true},
+      {overlay, [{copy, "scripts/brod", "bin"},
+        {copy, "{{lib_dirs}}/crc32cer/priv/crc32cer*.so", "bin"},
+        {copy, "{{lib_dirs}}/snappyer/priv/snappyer.so", "bin"}
+      ]}
+    ]}]},
+  {test, [
+    {deps, [ {docopt, {git, "https://github.com/zmstone/docopt-erl.git", {branch, "0.1.3"}}}
+           , {jsone, "1.7.0"}
+           , {meck, "0.9.2"}
+           , {proper, "1.4.0"}
+           , {snabbkaffe, "0.6.0"}
```

Possible to use the latest version? 
```suggestion
           , {snabbkaffe, "1.0.1"}
```

---

I tried to use the latest, but it caused some tests to fail.

---

Alright, let’s handle this change separately